src/interfaces/jira.ts

```diff
@@ -50,7 +50,7 @@ export interface JiraBuildData {
 }
 
 export interface JiraBranch {
-	createPullRequestUrl: string;
+	createPullRequestUrl?: string;
```

Comment:
This field is not mandatory https://developer.atlassian.com/cloud/jira/software/rest/api-group-development-information/#api-devinfo-0-10-bulk-post